### ISSUE TITLE
recipe_signatures_are_correct check is very slow

### ISSUE THREAD

**Issue by mythmon** [mozilla/normandy#900]
Below is a profile of running `./manage.py check` on stage. It shows that `recipe_signatures_are_correct` is taking 415 second to execute, which is not ok. Short term we are going to disable this check, but long term we should come up with something better. This could mean making this check faster, or possibly we could change the way we run checks to not run this in the request/response cycle of `/__heartbeat__`.

```
   ncalls  tottime  percall  cumtime  percall filename:lineno(function)
   1072/1    0.038    0.000  424.332  424.332 {built-in method builtins.exec}
        1    0.000    0.000  424.332  424.332 manage.py:2(<module>)
        1    0.000    0.000  422.955  422.955 __init__.py:362(execute_from_command_line)
        1    0.000    0.000  422.955  422.955 __init__.py:286(execute)
        1    0.000    0.000  418.192  418.192 base.py:277(run_from_argv)
        1    0.000    0.000  418.188  418.188 __init__.py:38(new_execute)
        1    0.000    0.000  418.188  418.188 base.py:308(execute)
        1    0.000    0.000  418.180  418.180 registry.py:64(run_checks)
        1    0.000    0.000  418.180  418.180 check.py:40(handle)
        1    0.000    0.000  418.180  418.180 base.py:363(check)
        1    0.000    0.000  418.180  418.180 base.py:360(_run_checks)
        1    0.032    0.032  415.497  415.497 checks.py:36(recipe_signatures_are_correct)
      179    0.005    0.000  401.481    2.243 signing.py:92(verify_signature)
      716    0.003    0.000  400.737    0.560 ellipticcurve.py:140(__rmul__)
 1259/901    2.027    0.002  400.735    0.445 ellipticcurve.py:109(__mul__)
   263608  222.149    0.001  382.215    0.001 numbertheory.py:185(inverse_mod)
   205710    7.816    0.000  310.590    0.002 ellipticcurve.py:149(double)
      179    0.044    0.000  278.231    1.554 keys.py:98(verify)
      179    0.012    0.000  278.183    1.554 keys.py:103(verify_digest)
      179    0.004    0.000  278.160    1.554 ecdsa.py:93(verifies)
```

CC @relud

**Comment by mythmon** [comment]
This test has been temporarily disabled, but we should probably work on improving the speed of this test and re-enable it.

**Comment by shell1** [comment]
this is fixed in #1820
